openstudiocore/src/openstudio_lib/ThermalZonesGridView.cpp

```diff
@@ -532,6 +532,9 @@ void ThermalZonesGridController::addColumns(std::vector<QString> & fields)
         [](model::ThermalZone * z) {
           if (boost::optional<model::ZoneControlHumidistat> thermostat = z->zoneControlHumidistat()) {
             thermostat->resetHumidifyingRelativeHumiditySetpointSchedule();
+            if( ! thermostat->dehumidifyingRelativeHumiditySetpointSchedule() ) {
```

Comment:
@kbenne @evanweaver correctly implemented this code https://github.com/NREL/OpenStudio/commit/656a7b5f547928dcd4591274e19f59125a8cbdfe to remove objects after a bad drop.  However, I think that in this case an empty humidistat can be created if you drop a bad schedule on the object.  That is if the set fails, we may need to check for empty humidity schedules and remove the humidistat.  Checking for this in the translator as a back up is probably a good idea too